### PR TITLE
Small tweaks in the quickstart guide

### PR DESCRIPTION
**Pull request by kolibril13**

### CODE REVIEW

File: docs/quickstart-myst-documents.md [hunk 1]
```diff
@@ -98,8 +98,8 @@ Unfortunately, this way of including this information is not easily machine-read
 
 A tutorial to evolve markdown documents and notebooks into structured data
 
-**Authors:** Rowan Cockett<sup>1,2</sup> \
-**Affiliations:** <sup>1</sup>Executable Books, <sup>2</sup> Curvenote \
+**Authors:** Rowan Cockett <sup>1,2</sup> \
+**Affiliations:** <sup>1</sup> Executable Books, <sup>2</sup> Curvenote \
```
Comment (kolibril13) [inline docs/quickstart-myst-documents.md:102]: In the according docs screenshot, the spacing could be updated as well :
image

**Comment by kolibril13** [comment]
One thought regarding the chapter dependencies.
Currently, the subchapter "Exporting to PDF" is dependent on the subchapter "Microsoft Word Documents" with the sentence `🛠 In 01-paper.md: replace export: docx with a list:` I think it would be nice to rephrase this sentence in a way that does not require the previous knowledge from the Microsoft Word Documents chapter, as people might skip that Microsoft Word chapter. 

* [Export Documents 📜](https://myst-tools.org/docs/mystjs/quickstart-myst-documents#export-documents)
    * [Microsoft Word Documents](https://myst-tools.org/docs/mystjs/quickstart-myst-documents#microsoft-word-documents)
     * [Exporting to PDF](https://myst-tools.org/docs/mystjs/quickstart-myst-documents#exporting-to-pdf)

**Comment by rowanc1** [comment]
So cool, thank you for these edits!!

What about:

```
🛠 In 01-paper.md: create an exports list with `docx` and `pdf` formats
```

Do you want to make that change or similar if you have it open?

**Comment by rowanc1** [comment]
Thanks @kolibril13, appreciate the fixes!! Absolutely would love the help on other docs pages as you are going through them, and on calling out with issues any places you get stuck as well!

🚀

**Comment by kolibril13** [comment]
I am happy that this is helpful! :)
One more thing:
here the abstract background is grey, but for me, it's not:
https://myst-tools.org/docs/mystjs/quickstart-myst-documents#use-myst-markdown

image

image

**Comment by rowanc1** [comment]
Yes -- that is a known bug, I took the screenshot on a branch of `@myst-theme` we are aiming to get in over the next week or so. The very experimental [article theme](https://github.com/executablebooks/myst-article-theme) is pushing the way here.

I will see if I can merge these changes in today, but if not I should probably fix the screenshots.

**Comment by rowanc1** [comment]
I opened up a bug to track that! #216

**Comment by kolibril13** [comment]
aand one more:
The emoji does not render in latex.
Maybe this can be noted somewhere in the quick start that imagemagick is needed, like a nice rephrasing of the error message
"To convert these images, you must install imagemagick or inkscape"
image

```
% myst build 01-paper.md


📬 Performing exports:
   01-paper.md -> _build/exports/paper.docx
   01-paper.md -> _build/exports/paper.pdf
📖 Built 01-paper.md in 41 ms.
📖 Built 02-notebook.ipynb in 41 ms.
📖 Built 01-paper.md in 36 ms.
📖 Built 02-notebook.ipynb in 36 ms.
⛔️ README.md Cannot convert SVG images, they may not correctly render.
   To convert these images, you must install imagemagick or inkscape
📖 Built README.md in 199 ms.
📚 Built 3 pages for project in 227 ms.
📑 Exported TeX in 2.52 ms, copying to _build/temp/mystlV4LyM/paper.tex
'parts' missing required key: abstract (at 01-paper.md)
'frontmatter' missing required key: short_title (at 01-paper.md)
🖨  Rendering PDF  (volcanica) to _build/temp/mystgBLlMl/paper.pdf
⛔️ README.md Cannot convert SVG images, they may not correctly render.
   To convert these images, you must install imagemagick or inkscape
📖 Built README.md in 285 ms.
📚 Built 3 pages for project in 306 ms.
📄 Exported DOCX in 88 ms, copying to _build/exports/paper.docx
📄 Exported PDF  (volcanica) in 6.8 s, copying to _build/exports/paper.pdf
```

**Comment by rowanc1** [comment]
Fixed up in #218.

docs-improvements

**Pull request merged**